x/axelarnet/module.go

```diff
@@ -344,6 +342,16 @@ func (m AxelarnetIBCModule) setRoutedPacketFailed(ctx sdk.Context, packet channe
 	// check if the packet is Axelar routed cross chain transfer
 	transferID, ok := getSeqIDMapping(ctx, m.keeper, port, channel, sequence)
 	if ok {
+		lockableAsset, err := m.nexus.NewLockableAsset(ctx, m.ibcK, m.bank, funcs.MustOk(m.keeper.GetTransfer(ctx, transferID)).Token)
+		if err != nil {
+			return err
+		}
+
+		err = lockableAsset.LockFrom(ctx, types.AxelarIBCAccount)
```

Comment:
I think this will break in-transit IBC transfers that timeout or have an ack error. Those packets were sent from the escrow accounts, and so the coin would be refunded back to the escrow and the locking here will fail as a result. We can explicitly check what the packet sender was to distinguish this behaviour. But could timeout/ack error be delayed indefinitely? In that case, we might always need to have this fallback